modules/tensor_mechanics/test/tests/smeared_cracking/cracking_power.i

```diff
@@ -0,0 +1,93 @@
+#
+# Simple pull test for cracking.
+# The stress increases for two steps and then drops to zero.
```

Comment:
It doesn't look like it will drop to zero.

Reply:
You're absolutely right...